docs/gallery-series-options.Rmd

```diff
@@ -46,13 +46,60 @@ dygraph(ldeaths, main = "Deaths from Lung Disease (UK)") %>%
   dyOptions(drawPoints = TRUE, pointSize = 2)
 ```
 
+Different point shapes are available as well:
```

Comment:
I think we only need one example plot drawn here. We can do that plus just enumerate the various available point shapes.